setup.py

```diff
@@ -74,10 +74,10 @@
             "numpy",
             "sympy",
             "pandas",
-            "jupytext>=1.6.0",
+            "jupytext-ebp==1.7.2a0",  # TODO waiting for jupytext 1.7.2 release
         ],
         "rtd": [
-            "jupytext>=1.6.0",
+            "jupytext-ebp==1.7.2a0",  # TODO waiting for jupytext 1.7.2 release
```

Comment:
I think this is now out but it is 1.8.0: https://github.com/mwouts/jupytext/releases/tag/v1.8.0

Reply:
Oh yeh I know 👍 https://github.com/mwouts/jupytext/pull/692